common/Helpers/WindowsOptionalFeatures.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using DevHome.Common.Environments.Helpers;
+
+namespace DevHome.Common.Helpers;
+
+public static class WindowsOptionalFeatures
+{
+    public const string Containers = "Containers";

```

Comment:
Do these need to be localized?

Reply:
No, these are the feature names "keys", not intended for display, just for querying or managing.